core/vm/opcodes.go

```diff
@@ -209,6 +209,7 @@ const (
 	CALLCODE
 	RETURN
 	DELEGATECALL
+	CREATE2
```

Comment:
Please skim through this file, there are 1 or 2 more instances where you need to add the opcode (the string to opcode and opcode to string mappings).